context/app/static/js/components/Detail/DagProv.jsx

```diff
@@ -32,5 +33,14 @@ function DagProv(props) {
     </>
   );
 }
+DagProv.propTypes = {
+  dagListData: PropTypes.arrayOf(
+    PropTypes.shape({
+      hash: PropTypes.string,
```

Comment:
should we have isRequired on the inside, too?

Reply:
I could switch `PropTypes.shape` to `PropTypes.exact`